### DISCUSSION TITLE
Consumption workflow - Survey and Observation( with and without rubric) created using KP for Manage Learn Usecase.

### DISCUSSION THREAD

**Discussion by aks30** [Sunbird-Knowlg/Community#30] [General Discussions]
We have started a thread in Sunbird InQuiry Building block but haven't got any comment or feedback or approval so mentioning it here as well in case the same belongs here - 

https://github.com/Sunbird-inQuiry/Community/discussions/18

Cc - @vijiurs, @kiranharidas187 @aishwaryashikshalokam, @priyanka-TL, @VISHNUDAS-tunerlabs, @Shakthieshwari, @Prateek-slokam, @MeghnaSL, @joffinjoy, @githubpiyush

**Comment by vinukumar-vs** [comment]
@aks30 On high level I went through the details, It is more specific to Questions, Question-set & Programs. Don't have much context of Observations, Survey & Solutions what you detailed in https://github.com/orgs/Sunbird-inQuiry/discussions/18.

Would suggest to the details it in compact & easy to understand. Break the requirement into multiple discussions so that it will be easy to respond & close.

I would suggest to have a detailed discussion/meeting to understand the your expectations & plan. Plese schedule a meeting to discuss.

@surendrasinghs @rhwarrier @vrayulu @maheshkumargangula
